src/sentry/api/serializers/models/organization_member.py

```diff
@@ -112,6 +112,7 @@ def serialize(
             "flags": {
                 "sso:linked": bool(getattr(obj.flags, "sso:linked")),
                 "sso:invalid": bool(getattr(obj.flags, "sso:invalid")),
+                "member-limit:restricted": bool(getattr(obj.flags, "member-limit:restricted")),
```

Comment:
Do we already have logic about membership limits as part of open source sentry. Should we name this something else?

Reply:
@EvanPurkhiser What's wrong with this name? I think we decided this name was OSS friendly.

Reply:
Just wanting to make sure we're OK with the idea of a member limit in open source. I can't remember if it has logic to do that in sentry.

My reason for asking is, this might imply to open source users that there's a way to restrict the number of members in their orgs, but if we don't have that functionality this just becomes a red herring.

Reply:
though tbh `deactivated` would probably be confusing too?

Reply:
@EvanPurkhiser We don't enforce a member limit in Sentry. There just aren't great solutions that don't incur heavy costs like adding another model to push this flag. Hard to balance keeping open source completely clean of these kinds of changes vs simplicity and performance of SaaS.

Reply:
For the record, Cramer said this was OK.

Reply:
Would it be OK to call it like `hard-deactivated` or something like that?

Reply:
@EvanPurkhiser probably not a great idea case we end up deactivating for other reasons I think

Reply:
In that case it would proably be better to have a deactivated reason

Reply:
which isn't just hypothetical (the billing team was going to do exactly that)

Reply:
Then we should definitely rename this to something like `hard-deactivated` with an enum, because you've already generically named your hooks like `disabled-member-tooltip`, which implies sentry will be responsible for deciding what to show as a reason.

Reply:
As discussed over PM, we are already using the flag in auth and getsentry so renaming the flag is a complicated migration with many steps. Not worth it just because the name isn't great :)